eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -49,6 +49,7 @@ jobs:
       - template: /eng/pipelines/templates/steps/install-dotnet.yml
         parameters:
           NuGetCacheKey: Test
+      - script: 'dotnet dev-certs https --trust'
```

Comment:
Does this need to happen for all pipelines?

Reply:
no, just MacOS - but this was just a quick and dirty test.